app/cells/decidim/vocdoni/voting_step_navigation/show.erb

```diff
@@ -1,29 +1,28 @@
-<div class="focus__footer">
-  <div class="row">
-    <% if first_step? %>
-      <a></a>
-    <% else %>
-      <%= link_to(
-        icon("chevron-left", class: "icon", role: "img", "aria-hidden": true) + " " + t("decidim.vocdoni.votes.voting_step.back"),
-        "#",
-        class: "focus__back",
-        data: {
-          toggle: [previous_step_dom_id, current_step_dom_id].join(" ")
-        }
-      ) %>
-    <% end %>
-
-    <span class="button focus__next disabled-continue disabled">
-      <%= button_continue_text %>
-    </span>
-
+<%= append_javascript_pack_tag "decidim_vocdoni_voter_new_vote" %>
+<div class="form__wrapper-block flex-col-reverse md:flex-row justify-between">
+  <% if first_step? %>
+    <a></a>
```

Comment:
this looks strange (I know that was there before), but do you know the reason why we put this empty link here?